recipes/publish/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.1" %}
+{% set hg_tag = "164:b7f90dcfb235" %}
+
+package:
+  name: publish
+  version: {{ version }}
+
+source:
+  hg_url: https://bitbucket.org/logg/publish
+  hg_tag: {{ hg_tag }}
```

Comment:
Here is the bug report I made about this: https://bitbucket.org/logg/publish/issues/5/please-tag-the-current-version